.circleci/config.yml

```diff
@@ -425,6 +425,7 @@ jobs:
       - checkout
       - pip-install
       - run: ruff check
+      - run: ruff check --preview --select=E303
```

Comment:
Is there any way to have `ruff check` include E303 whenout adding these extra command line argument?

I'm just thinking for local development its nice to just run `ruff check` from time to time.

Reply:
Nope.  We need to target using both `--preview` and `--select=E303`.  Without `--preview`, `E303` will not be tested.  Without `--select=E303` then `ruff check --preview` with our current ruff config in `pyproject.toml` returns:
```
Found 17355 errors.
[*] 497 fixable with the `--fix` option (221 hidden fixes can be enabled with the `--unsafe-fixes` option).
```

I chose this approach because running both commands one after the other will take a small fraction of one second.

Reply:
Ok that sounds like a reasonable solution for now.  Can you add a comment explaining why this is needed?   Is it simply that the ruff authors are still working on the E303 rule?  Will it become stable at some point and we can remove this?  Perhaps mention that in the comment?

Reply:
Will do… https://docs.astral.sh/ruff/preview